### DISCUSSION TITLE
Looking for Feedback - New Design

### DISCUSSION THREAD

**Discussion by marcelfolaron** [Leantime/leantime#698] [Ideas]
Hi Community,

I am working on freshening up the design a bit and cleaning up the CSS. Here is an updated design draft. 
What do you think?
Anything you'd like to see from a new design? 

<img width="1215" alt="new-design" src="https://user-images.githubusercontent.com/47038998/166146146-de6d0423-5e04-49ca-a497-438cbbdd5a24.png">

PS: As part of the css update I am moving all colors into a new var css files that will allow you to change the entire theme of the site.

**Comment by joaobrazuna** [comment]
Nice 👍

**Comment by thepatchbay** [comment]
It looks cleaner, I am a fan of the rounded corner design.  dedicated settings panel for colours would be awesome. Even better if here was a CSS box to add custom CSS to classes.

**Comment by axelrindle** [comment]
Looking good. What about adding support for [Bootswatch Themes](https://bootswatch.com/) ?

**Reply by marcelfolaron** [reply]
Thanks. Unfortunately we have customized the theme so much that I'm afraid we wouldn't be able to utilize bootswitch themes. 
I'm trying to add as many customizations as possible and maybe even theme templates.

**Comment by unknown797** [comment]
Hi Marcel,

Love your work as always.

Current / existing design (2.1.9):

- Prefer the square sharp corners over rounded (just feels tidy) for the columns (New, Blocked, In Progress etc..), I think it makes it stand out more rather than box within box feel.
- Once again prefer the square priority, New milestones etc.. button / drop down just feels tidier.
- Column headers, being blue with the underline green, blue etc.. is subtle which is nice (anyway can add a custom setting to change it if we need to? at the moment only label is allowed to be changed.).
- Love side menu being dark makes it standout more over white.

New design (feedback):

- Love the use of colours for priorities (Medium, High and Critical)
- Not sure if mile store boxes requires colour as its quite distracting (might just need to be grey for no milestones and no priority as above?)
- Love the flat avator icon where you can assign someone, I think that icon looks nicer compared to the current 3D one.
- Unsure on the trimming of the column boxes, I think your current version looks cleaner as it draws it to the bottom of the screen, where your new version only expands when there is a todo card, which has more white space. (Could look good over time just initially it looks empty and having the border to the bottom makes more sense?)
- The colour matching the priority on the left looks awesome.
- Is it possible to remove the tasks heading? Seems odd that in the todo list we have a []task on every card?

Hope this helps and thanks for sharing your initial design and asking for feedback.

**Reply by marcelfolaron** [reply]
Thank you, I appreciate the detailed feedback!

**Comment by alexcamlo** [comment]
Hi, 
I have created a proposal for the ToDo Kanban view basing the design in a cleaner look as in the screenshot of the first post.
If it is a path you would like to keep exploring I could create a UI library in Figma or any other communication media with styles for the basic components (buttons, forms, text styles, etc.)

![Leantime-ToDosKanban](https://user-images.githubusercontent.com/121455/177008814-4a54e6d1-f686-4cd9-bf44-1d2ebd509950.png)

**Reply by gloriafolaron** [reply]
I really like how clean this is.  @marcelfolaron for visibility

**Reply by alexcamlo** [reply]
I have been playing around with other layouts. In this case, merging the blue header with the sidebar to give more vertical space to the content.
![Desktop - No header - User down](https://user-images.githubusercontent.com/121455/179345836-92d6f7d0-9ba6-49e7-a373-d802bfde1d13.png)
![Desktop - No header - User up](https://user-images.githubusercontent.com/121455/179345839-c5f5645d-9f1d-4280-9385-2ce4ccbbd09b.png)

**Comment by root9191** [comment]
Will there also come a Dark Mode? 😎

**Comment by marcelfolaron** [comment]
Already on it :)
